android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -878,6 +878,26 @@ public void onFailure(@NonNull Exception exception) {
           result.success(null);
           break;
         }
+
+      case "style#sourceExists":
+        {
+          if (style == null) {
+            result.error(
+                "STYLE IS NULL",
+                "The style is null. Has onStyleLoaded() already been invoked?",
+                null);
+          }
```

Comment:
add a break in the body of the if or line 892 will throw

Reply:
i'll make the change

Reply:
still seems to be missing?